src/routes/api/submissions/+server.ts

```
@@ -109,6 +109,13 @@ export const PATCH: RequestHandler = async ({ request, cookies, url }) => {
     // get pr item
     const pr = await items.getOne({ id: body!.item_id });
     if (pr) {
+      const createdAtDate = isNaN(new Date(body!.created_at as string).getTime())
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)